presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -4621,15 +4621,21 @@ private void eraseStatistics(SchemaTableName schemaTableName)
     /**
      * @return query id
      */
-    private String insertData(SchemaTableName tableName, MaterializedResult data)
+    protected String insertData(SchemaTableName tableName, MaterializedResult data)
+            throws Exception
+    {
+        return insertData(tableName, data, Optional.empty());
```

Comment:
nit: Since we're overloading the method. can we pass newSession() here and then ConnectorSession doesn't need to be optional?